cmd/s2i/main.go

```diff
@@ -171,6 +171,7 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVarP(&(cfg.Ref), "ref", "r", "", "Specify a ref to check-out")
 	buildCmd.Flags().StringVarP(&(cfg.AssembleUser), "assemble-user", "", "", "Specify the user to run assemble with")
 	buildCmd.Flags().StringVarP(&(cfg.ContextDir), "context-dir", "", "", "Specify the sub-directory inside the repository with the application sources")
+	buildCmd.Flags().StringVarP(&(cfg.ExcludeRegExp), "exclude-regex", "", "", "Regular expression string for selecting which files from the application source tree to exclude from the tar stream")
```

Comment:
i think we can just call this "exclude".  How are multiple patterns handled?  comma separated list?  if so the help should indicate that.